### ISSUE TITLE
How can I cancel a operation running in the "Output" tab of vscode like the "Upload" operation

### ISSUE THREAD

**Issue by lefuturiste** [microsoft/vscode-arduino#1078]

**Comment by hellyzh** [comment]
Terminate the related process of Arduino IDE command using task manager can abort the current running operation.

**Comment by lefuturiste** [comment]
yes but it's a not very practical way to do it but I guess this is the only way to do it for now. So this is more like a "enhancement" kind of issue, a feature request
